zuul-core/src/main/java/com/netflix/zuul/netty/server/ZuulServerChannelInitializer.java

```diff
@@ -28,12 +28,24 @@
  */
 public class ZuulServerChannelInitializer extends BaseZuulChannelInitializer {
 
-    public ZuulServerChannelInitializer(int port,
-                                        ChannelConfig channelConfig,
-                                        ChannelConfig channelDependencies,
-                                        ChannelGroup channels)
-    {
-        super(port, channelConfig, channelDependencies, channels);
+    public ZuulServerChannelInitializer(
+            String metricSuffix,
+            ChannelConfig channelConfig,
+            ChannelConfig channelDependencies,
+            ChannelGroup channels) {
+        super(metricSuffix, channelConfig, channelDependencies, channels);
+    }
+
+    /**
+     * Use {@link #ZuulServerChannelInitializer(String, ChannelConfig, ChannelConfig, ChannelGroup)} instead.
+     */
+    @Deprecated
+    public ZuulServerChannelInitializer(
+            int port,
+            ChannelConfig channelConfig,
+            ChannelConfig channelDependencies,
+            ChannelGroup channels) {
+        this(String.valueOf(port), channelConfig, channelDependencies, channels);
```

Comment:
Maybe add a convenience constructor to `BaseZuulChannelInitializer` that takes an `int`?

Reply:
I'm trying to stamp out all uses of the int param, so passing the string makes it easier for me to see that it's correct.   I would prefer updating this later, since it lets me ensure all the correct places have been updated.